src/components/MovieCard/MovieCard.tsx

```diff
@@ -3,42 +3,44 @@ import { Movie } from '../../types/Movie';
 import './MovieCard.scss';
 
 type Props = {
-  movie: Movie,
+  movie: Movie | null,
```

Comment:
Are you sure that movie can be null? 
You render MovieCard only if movie is present, so movie won't be null anyway
![image](https://github.com/mate-academy/react_movies-list-fetch-movies/assets/96409563/c29c0456-6788-470f-8ccb-fa97fbb2a4a8)